pkgs/applications/virtualization/virt-viewer/default.nix

```diff
@@ -22,8 +22,9 @@ stdenv.mkDerivation rec {
   buildInputs = [
     pkgconfig intltool glib libxml2 gtk3 gtkvnc gmp libgcrypt gnupg cyrus_sasl
     shared_mime_info libvirt libcap_ng yajl gsettings_desktop_schemas makeWrapper
-    xen numactl
-  ] ++ optionals spiceSupport [ spice_gtk spice_protocol libcap gdbm ];
+    numactl
+  ] ++ optionals spiceSupport [ spice_gtk spice_protocol libcap gdbm
+  ] ++ optional stdenv.is64bit xen;
```

Comment:
Actually xen seems to run on both x86 and amd64, but I'm not using it so cannot say why the ```platforms``` limitation was put in place.